docs/data-ai/ai/run-inference.md

```diff
@@ -43,7 +43,7 @@ Then, from the **Select model** dropdown, select the name of the ML model servic
 
 ### Test your changes
 
-You can test a deployed vision service by clicking on the **Test** area of its configuration panel or from the [**CONTROL** tab](/manage/troubleshoot/teleoperate/default-interface/#viam-app).
+You can test a deployed vision service by clicking on the **Test** area of its configuration panel or from the [**CONTROL** page](/manage/troubleshoot/teleoperate/default-interface/#viam-app).
```

Comment:
This is probably worth discussing as a team. We historically refer to it as a tab. It used to look more like a tab. You can leave the change but worth bringing up to the next team sync

Reply:
Added to our weekly sync doc as a discussion item. I started to do this because some parts of the docs already use 'page' and I found 'DATA tab's DATASET subtab' kind of awful. But I suspect y'all have put more thought into this than me and I should leverage that.

Reply:
Agreed subtab used to be rough, but that was the next level down when we had configure tab, components subtab. I think **DATA** page, **CONTROL** tab and **DATASET** tab works. Data is at the level of fleet, so it can be a page I think haha.

Reply:
(Also I am only looking at this because I was asked to; not just randomly double-reviewing!)